### ISSUE TITLE
Move to `rust-bitcoin` tracking issue

### ISSUE THREAD

**Issue by tcharding** [apoelstra/rust-jsonrpc#78]
This issue is to track any remaining tasks before we can move this repo over to the [`rust-bitcoin`](https://github.com/rust-bitcoin) organisation.

What tasks are remaining?

**Comment by apoelstra** [comment]
I think we're good to go. We have
* active maintenance of the libarry
* clean clippy runs and 1.41 builds, CI-enforced, as well as a fuzz harness (not yet in CI)
* its maintainers (me, you) are active in the rust-bitcoin community

There's still a fair bit more to do with this library but I'd say that it's in good enough shape to be in rust-bitcoin

**Comment by tcharding** [comment]
BOOM!

**Comment by tcharding** [comment]
I think we should move this crate after merging my recent flurry of PRs

  - #117 
  - #119 
  - #120 
  - #121 
  - #123

**Comment by tcharding** [comment]
What if we move the new `rust-bitcoind-json-rpc` to `rust-bitcoin` org and put `jsonrpc` crate inside that repo? Its the same reviewers and the same domain.

**Comment by apoelstra** [comment]
Yeah, I like that idea. Then we have all the "crates that we don't like but maintain out of honour" are in one repo.

**Comment by tcharding** [comment]
lolz, exactly.
